Should I be able to use optimized_route with pyvalhalla?

If I use the the following code:

```
from valhalla import Actor, get_config
from valhalla.utils import decode_polyline

config = get_config(tile_dir='./custom_tiles/valhalla_tiles', verbose=True)
# instantiate Actor to load graph and call actions
actor = Actor(config)

location = actor.optimized_route({"directions_options":{"units":"miles"}, "costing": "auto", "locations":[{"lat":-31.953512,"lon":115.857048}, {"lat": -32.536104, "lon": 115.727539},{"lat": -32.056946, "lon": 115.727539},{"lat": -31.953512, "lon": 115.857048}]})
print(location)
```

I get back the following error:

```
[I] ➜ python test.py
2023/04/08 17:16:39.296473 [ERROR] (stat): /Users/juda/GitHub/PinDrop/RouteOptimisation/valhalla_tiles.tar No such file or directory
2023/04/08 17:16:39.297796 [WARN] Tile extract could not be loaded
2023/04/08 17:16:39.298081 [WARN] (stat): /data/valhalla/traffic.tar No such file or directory
2023/04/08 17:16:39.298087 [WARN] Traffic tile extract could not be loaded
Traceback (most recent call last):
  File "test.py", line 8, in <module>
    location = actor.optimized_route({"directions_options":{"units":"miles"}, "costing": "auto", "locations":[{"lat":-31.953512,"lon":115.857048}, {"lat": -32.536104, "lon": 115.727539},{"lat": -32.056946, "lon": 115.727539},{"lat": -31.953512, "lon": 115.857048}]})
TypeError: optimized_route(): incompatible function arguments. The following argument types are supported:
    1. (self: valhalla._valhalla._Actor, arg0: str) -> str

Invoked with: <valhalla.actor.Actor object at 0x108f5c4a0>, {'directions_options': {'units': 'miles'}, 'costing': 'auto', 'locations': [{'lat': -31.953512, 'lon': 115.857048}, {'lat': -32.536104, 'lon': 115.727539}, {'lat': -32.056946, 'lon': 115.727539}, {'lat': -31.953512, 'lon': 115.857048}]}
```
If I replace optimised_route with straight up route, everything works perfectly.  I noticed that actor.py does not define optimized_route, but it does look like it's plumbed in _valhalla.cc.

I've tried making changes locally to submit a PR, but the build is failing miserably on OSX so I can't test it.

Thanks for the work on this, it's great!

--- a/valhalla/actor.py
+++ b/valhalla/actor.py
@@ -80,6 +80,10 @@ class Actor(_Actor):
         return super().route(req)
 
     @dict_or_str
+    def optimized_route(self, req: Union[str, dict]):
+        return super().route(req)
+
+    @dict_or_str
     def locate(self, req: Union[str, dict]):
         return super().locate(req)
 

